crates/nu-command/Cargo.toml

```diff
@@ -134,7 +135,53 @@ features = [
 workspace = true
 
 [features]
-plugin = ["nu-parser/plugin"]
+default = ["os"]
+os = [
+	# include other features
+	"js",
+	"network",
+	"nu-protocol/os", 
+	"nu-utils/os",
+
+	# os-dependant dependencies
+	"crossterm",
+	"notify-debouncer-full",
+	"open",
+	"os_pipe",
+	"uu_cp",
+	"uu_mkdir",
+	"uu_mktemp",
+	"uu_mv",
+	"uu_touch",
+	"uu_uname",
+	"uu_whoami",
+	"which",
+]
+
+# The dependencies listed below need 'getrandom'. 
+# They work with JS (usually with wasm-bindgen) or regular OS support. 
+# Hence they are also put under the 'os' feature to avoid repetition.
+js = [
+	"getrandom", 
+	"getrandom/js", 
```

Comment:
Is `getrandom/js` necesssary for `os`?

Reply:
It's not but `os` should get `rand` which is not in `network`. Also `getrandom/js` is a noop for systems that have an OS. See [the docs of `getrandom`](https://docs.rs/getrandom/latest/getrandom/#webassembly-support), about the `js` feature:
> This feature has no effect on targets other than `wasm32-unknown-unknown`.